enclave-manager/web/src/client/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const KURTOSIS_DEFAULT_HOST = "localhost";
+export const KURTOSIS_DEFAULT_PORT = 8081;
+export const KURTOSIS_DEFAULT_URL = `http://${KURTOSIS_DEFAULT_HOST}:${KURTOSIS_DEFAULT_PORT}`;
```

Comment:
Is there a way we can have different ones for different environments? We have that in the cloud ui.

Reply:
not sure what you mean - I've actually just used similar names to what's in the current engine/frontend - maybe it would be clearer if I just moved these constants into `LocalKurtosisClient`

Reply:
For reference, check out the `.env*` files in the cloud ui repo